finetrainers/args.py

```diff
@@ -234,6 +251,12 @@ def _add_model_arguments(parser: argparse.ArgumentParser) -> None:
         default=None,
         help="The directory where the downloaded models and datasets will be stored.",
     )
+    parser.add_argument("--text_encoder_dtype", type=str, default="bf16", help="Data type for the text encoder.")
+    parser.add_argument("--text_encoder_2_dtype", type=str, default="bf16", help="Data type for the text encoder 2.")
+    parser.add_argument("--text_encoder_3_dtype", type=str, default="bf16", help="Data type for the text encoder 3.")
+    parser.add_argument("--transformer_dtype", type=str, default="bf16", help="Data type for the transformer model.")
+    parser.add_argument("--unet_dtype", type=str, default="bf16", help="Data type for the U-Net model.")
```

Comment:
Do we have any `unet`-based model? :D

Reply:
I do have a local branch for training SD1.5 :) Helps test things faster

Reply:
Since the arg list already quite big maybe it won't hurt to try to remove arguments we don't want to support models we don't want to thoroughly test. WDYT? No strong opinions.

Reply:
Could you tell me which ones are not useful and can be removed?

Reply:
`unet_dtype` as it’s ot relevant for the models we currently want to support.

Reply:
Removed